docs/README.md

```diff
@@ -52,7 +52,7 @@ Where \<yy.mm\> is the version of Triton that you want to pull. For a complete l
 This guide covers the simplest possible workflow for deploying a model using a Triton Inference Server.
 - [Create a Model Repository](getting_started/quickstart.md#create-a-model-repository)
 - [Launch Triton](getting_started/quickstart.md#launch-triton)
-- [Send an Inference Request](getting_started/quickstart.md#sending-an-inference-request)
+- [Send an Inference Request](getting_started/quickstart.md#send-an-infernce-request)
```

Comment:
```suggestion
- [Send an Inference Request](getting_started/quickstart.md#send-an-inference-request)
```
Can you fix this typo in the header it points to as well? https://github.com/triton-inference-server/server/blob/main/docs/getting_started/quickstart.md#send-an-infernce-request